### ISSUE TITLE
Enhance formatting of long targets

### ISSUE THREAD

**Issue by beevee** [moira-alert/web#31]
Blocks of text like this on main page:

![2016-05-07_12-42-00](https://cloud.githubusercontent.com/assets/1009675/15090906/8d7039aa-1459-11e6-8623-eccab7f81c80.png)

...is a complete unreadable mess. We should do something, maybe format with newlines and spaces?

**Comment by gmlexx** [comment]
@melnikk, could you attach screenshot after your changes?

**Comment by melnikk** [comment]
![image](https://cloud.githubusercontent.com/assets/4337939/16793655/0297a38c-48ec-11e6-93ae-dc2bac57730f.png)

**Comment by beevee** [comment]
![2016-07-13_12-39-42](https://cloud.githubusercontent.com/assets/1009675/16795467/4a08f668-48ff-11e6-8f77-4b108724ba4c.png)
Can you also fix the trigger details page?

**Comment by melnikk** [comment]
![image](https://cloud.githubusercontent.com/assets/4337939/16795638/f37f8232-48f7-11e6-9516-93ca42baa445.png)
Fixed in 1153bd1

**Comment by beevee** [comment]
OK, we have two minor remarks:
1. We'd like to see function arguments on the same indent level. More like Clojure-style, see example below. Break after closing bracket, but not before.
2. If function arguments are long strings, put them on separate lines (like divideSeries arguments in the example). If they are short, put them on a single line (like 78, 100 in the example).

```
aliasByNode(
    movingAverage(
        scale(
            divideSeries(
                DevOps.rabbitmq.vm-elk2.health.mem_used,
                DevOps.rabbitmq.vm-elk2.health.mem_limit),
            78, 100),
        10),
    2)
```

I think, second remark will require some refactoring - so let's see how you handle this :)

**Comment by melnikk** [comment]
![image](https://cloud.githubusercontent.com/assets/4337939/16803753/9cf14dd6-4921-11e6-9ae8-e35a57922bed.png)
Done in aa454ee

**Comment by beevee** [comment]
Seems like you have an extra space at the beginning of args lines?
" 2)", " 2, 4)", etc?

**Comment by melnikk** [comment]
Fixed in fd0044b :)

**Comment by beevee** [comment]
One more bug: you shouldn't break on commas {inside,curly,braces}. They are a part of a pattern, not separate arguments.

Maybe it's time to write some tests? We can provide examples of patterns.

**Comment by melnikk** [comment]
Ok. I'm waiting for patterns. To be continued tomorrow...

**Comment by beevee** [comment]
Here you go: https://gist.github.com/beevee/a51c83319b76fc6e216f8885d08a5f27

Please, write a test for your code using these examples.

**Comment by beevee** [comment]
Any news?

**Comment by melnikk** [comment]
In progress.

**Comment by melnikk** [comment]
@beevee Curly braces fixed in 8c5bb5a
Asked for test details in gmail.
